Dockerfile

```
@@ -3,6 +3,10 @@
 # https://github.com/docker-library/repo-info/tree/master/repos/alpine/remote
 FROM alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad
```

We need to add `--platform` for this to work and make sure we are using the main image instead of the arch-specific one.

According to https://github.com/docker-library/repo-info/tree/master/repos/alpine/remote the latest 3.16.x is 3.16.3


```suggestion
FROM --platform=$BUILDPLATFORM alpine@sha256:b95359c2505145f16c6aa384f9cc74eeff78eb36d308ca4fd902eeeb0a0b161b
```

---

https://github.com/elastic/apm-aws-lambda/pull/362/commits/2a93275a70640d12ba76d8e80c3ebdbd18fe62a9